file_organizer_v2/src/file_organizer/services/auto_tagging/tag_learning.py

```
@@ -0,0 +1,513 @@
+
+        Returns:
+            List of (tag, confidence) tuples
+        """
+        predictions = {}
+
+        file_ext = file_path.suffix.lower()
+        directory = str(file_path.parent)
+
+        # Predict based on file type
+        if file_ext in self.file_type_tags:
+            for tag, count in self.file_type_tags[file_ext].most_common(10):
```

Ruff (0.14.13)

Loop control variable `count` not used within loop body

Rename unused `count` to `_count`

(B007)